feature: Add page Subtitle

YauheniKapliarchuk opened the pull request:
Changes:
- remove kicker;
- add page subtitle;
- update languages for Page Caption.

Commit:
feature: Add page Subtitle

Changes:
- remove kicker;
- add page subtitle;
- update languages for Page Caption.

hans-olson reviewed (changes requested):
Please revert back based on the discussion.

Commit:
Revert "feature: Add page Subtitle"

This reverts commit 244a79bb0a1c760152d9ffd04cf4e55884d9f744.

Commit:
feature: Add page Subtitle

Changes:
- add page subtitle;

hans-olson commented:
@daniel-eesti

I hope you can meet with us tomorrow to review this.

hans-olson commented:
<img width="1113" alt="Screenshot 2023-12-13 at 10 45 32" src="https://github.com/adventistchurch/alps-wordpress/assets/61567403/24bb526b-8dff-4424-8d0d-e42e44d17963">

claytonk committed:
Added lodash.js dependency to carbon fields initialization

Commit:
feature: Add page Subtitle Refactoring and Final Decision

Changes:
- Add Page subTitle;
- Integrate Page SubTitle in the current structure.

YauheniKapliarchuk committed:
Merge branch 'v3.1' into feature/add_page_subtitle

YauheniKapliarchuk commented:
Hi @hans-olson 

Current structure:

<img width="1499" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBekdMQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--55f7ee72b05e505cefb9d58ef1d34d396aa862d3/Screenshot%202023-12-28%20at%203.10.44%E2%80%AFPM.png" alt="Screenshot 2023-12-28 at 3.10.44 PM.png" />


Thanks,
Best Regards!

hans-olson commented:
I can not see the image. Can you upload again.

Commit:
feature: Add page Subtitle Refactoring and Final Decision

Changes:
- Change ordering.
